cves/kernel/CVE-2012-2745.yml

```
@@ -75,7 +71,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [833428]
```

I noticed that this appears to be the bug ID leading to bugzilla.redhat.com, is there a bug ID for this on bugzilla.kernel.org?
If not, I might suggest putting this for clarity:
```suggestion
bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=833428"]
```

---

this is a good catch, i forgot there was a difference between the ids for redhat and kernel